ui/src/styles.css

```
@@ -0,0 +1,32 @@
+/* MagnetarPrometheus UI design token baseline. */
+:root {
+  --mp-font-family-base: Inter, 'Segoe UI', sans-serif;
```

Stylelint (17.6.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)